camp_fin/static/js/chart_helper.js

```diff
@@ -14,434 +21,476 @@ ChartHelper.donations = function(el, title, sourceTxt, yaxisLabel, data, pointIn
+  el,
+  title,
+  sourceTxt,
+  yaxisLabel,
+  data
+) {
+  var color = "#007E85";
 
-  var seriesData = [{
-          color: color,
-          data: data[0],
-          name: "Donations and loans"
-        },{
-          color: "#DD0000",
-          data: data[1],
-          name: "Expenditures"
-        }
-      ]
+  var seriesData = [
+    {
+      color: color,
+      data: data[0],
+      name: "Donations and loans",
+    },
+    {
+      color: "#DD0000",
+      data: data[1],
+      name: "Expenditures",
+    },
+  ];
 
   //$("#charts").append("<div class='chart' id='chart_grouping_" + iteration + "'></div>")
   return new Highcharts.Chart({
-      chart: {
-          renderTo: el,
-          type: "area",
-          marginRight: 10,
-          marginBottom: 25
-      },
-      legend: {
-        backgroundColor: "#ffffff",
-        borderColor: "#cccccc",
-        floating: true,
-        verticalAlign: "top"
-      },
-      credits: {
-        enabled: false
-      },
+    chart: {
+      renderTo: el,
+      type: "area",
+      marginRight: 10,
+      marginBottom: 25,
+    },
+    legend: {
+      backgroundColor: "#ffffff",
+      borderColor: "#cccccc",
+      floating: true,
+      verticalAlign: "top",
+    },
+    credits: {
+      enabled: false,
+    },
+    title: null,
+    xAxis: {
+      dateTimeLabelFormats: { year: "%Y" },
+      type: "datetime",
+    },
+    yAxis: {
       title: null,
-      xAxis: {
-          dateTimeLabelFormats: { year: "%Y" },
-          type: "datetime"
-      },
-      yAxis: {
-          title: null
-      },
-      plotOptions: {
-        line: {
-          animation: false
-        },
-        series: {
-          marker: {
-            fillColor: color,
-            radius: 0,
-            states: {
-              hover: {
-                enabled: true,
-                radius: 5
-              }
-            }
-          },
-          shadow: false
-        }
-      },
-      tooltip: {
-          crosshairs: true,
-          formatter: function() {
-            var s = "<strong>" + ChartHelper.toolTipDateFormat("day", this.x) + "</strong>";
-            $.each(this.points, function(i, point) {
-              s += "<br /><span style='color: " + point.series.color + "'>" + point.series.name + ":</span> $" + Highcharts.numberFormat(point.y, 0, '.', ',');
-            });
-            return s;
+    },
+    plotOptions: {
+      line: {
+        animation: false,
+      },
+      series: {
+        marker: {
+          fillColor: color,
+          radius: 0,
+          states: {
+            hover: {
+              enabled: true,
+              radius: 5,
+            },
           },
-          shared: true
-      },
-      series: seriesData
-    });
-  }
+        },
+        shadow: false,
+      },
+    },
+    tooltip: {
+      crosshairs: true,
+      formatter: function () {
+        var s =
+          "<strong>" +
+          ChartHelper.toolTipDateFormat("month", this.x) +
```

Comment:
Looks like some formatting got applied. This is the only actual change in this file.

Reply:
(Changed from day to month)